config/src/config/consensus_config.rs

```diff
@@ -19,14 +19,14 @@ pub struct ConsensusConfig {
 impl Default for ConsensusConfig {
     fn default() -> ConsensusConfig {
         ConsensusConfig {
+            contiguous_rounds: 2,
```

Comment:
btw do you know why we have this as part of the consensus config o_X ?

Reply:
pre-dates me, I think there is some change in the guarantees by shrinking this or increasing it, but without thinking very hard.... @zekun000 / @aching do you want to eliminate this variable?

Reply:
It's the number of contiguous proposal rounds for a rotating proposer.  E.g. proposer X does rounds Y and Y+1, proposer X+1 does rounds Y+2 and Y+3.  It isn't used in the current LeaderReputation mode, but something we might want to in the future.

Reply:
Oh, in that case, it should be part of the proposer type or completely eliminated (as it isn't being used :))